code/__DEFINES/dcs/signals.dm

```diff
@@ -127,8 +127,10 @@
 // /turf signals
 #define COMSIG_TURF_CHANGE "turf_change"						//from base of turf/ChangeTurf(): (path, list/new_baseturfs, flags, list/transferring_comps)
 #define COMSIG_TURF_HAS_GRAVITY "turf_has_gravity"				//from base of atom/has_gravity(): (atom/asker, list/forced_gravities)
-#define COMSIG_TURF_MULTIZ_NEW "turf_multiz_new"				//from base of turf/New(): (turf/source, direction)
 */
+#define COMSIG_TURF_MULTIZ_DEL "turf_multiz_del"
+///from base of turf/multiz_turf_new: (turf/source, direction)
+#define COMSIG_TURF_MULTIZ_NEW "turf_multiz_new"
```

Comment:
these need to be implemented 
tl;dr
copypasting the defines in don't mean anything if they're not fired using SEND_SIGNAL 
find where it's doing this in virgo and send hte signal, or anything using this won't ever receive them

Reply:
Signals defined.